src/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -106,6 +106,19 @@ installChannels() {
       <% }) -%>
     <% }) -%>
   <% } -%>
+  
+  # Restart Explorer 
+  printItalics "Restarting Explorer" "U1F504"
```

Comment:
I think this line can be removed - we already have logs about restarting global and org explorer, and this particular line gets printed even if explorer is disabled